### ISSUE TITLE
Update for latest autogen and memgpt

### ISSUE THREAD

**Issue by nickbit** [tylerprogramming/ai#1]
Hi Tyler, 
You may need to update the files to comply with the latest changes of `autogen` and `memgpt`.
For example, `create_autogen_memgpt_agent` has a changed signature and does not accept a `persona` argument anymore.
Thank you for your great work.

**Comment by tylerprogramming** [comment]
@nickbit @CynicalWilson okay thank you for letting me know, I will check this out tomorrow and look at their repo to see what changed.  👍

**Comment by tylerprogramming** [comment]
Just quickly looking at it (not at home) it looks like they have a system_message as a parameter now, and that gets converted to a persona inside the function call.  And it’s persona now not persona_desc.  Yeah they definitely changed that, I think to make it look more like AutoGen.  I’ll have an update when I get on my computer tomorrow

**Comment by tylerprogramming** [comment]
I added an issue on memgpt: https://github.com/cpacker/MemGPT/issues/568

I updated code locally to work-ish.  For just GPT and not local, and also just AutoGen and not MemGPT agents, it works.  However as soon as I set it True to use a MemGPT agent I get a 401 for the openai API.  After debugging through it I see that it doesn't take the api_key for the memgpt configuration.  I see where it happens after debugging, and I've tried to hard-set it a few ways but couldn't get it to work.  I opened an issue ticket with a link above.  I will get back to this again as soon as I can.  I will update code and push it, but just know it may not work just yet.  Just want to see what they respond with or if somebody else faced the same issue.
